mycroft/configuration/config.py

```diff
@@ -230,15 +230,14 @@ def load_config_stack(configs=None, cache=False):
             return base
 
     @staticmethod
-    def init(ws):
```

Comment:
I changed all the core uses of this.  Are there other repos that could call this?

Reply:
Technically things like third party enclosures may use it or other things build for working with mycroft-core.

I do think we can have a rather short depreciation period though and remove it outright in 19.08 since it's a rarely touched thing.

Reply:
OK.  will add an init method back that calls the new method and add a deprecation warning.

Reply:
ugh!  since it is a static method, I had to actually duplicate the method rather than have one call the other.  I added a deprecation message.  How do we track things like this that will be breaking changes in 19.08?  Do we just search the code base for comments that mention 19.08?

Reply:
That's basically it yeah `TODO: 19.08` or similar comments